Automattic/jetpack
# Boost: Fix fatal error when not specifying a valid module to Boost's CLI

Fixes #36670

## Proposed changes:

* Fix fatal error when using `wp jetpack-boost module activate`;
* Update error message to show list of available modules.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






### Test that it breaks
* Run `jetpack docker wp jetpack-boost module activate` to make sure there's an error:

```
Error: There has been a critical error on this website.Learn more about troubleshooting WordPress. There has been a critical error on this website.
```

### Test that it works
* Switch to this branch;
* Run `jetpack docker wp jetpack-boost module activate` to see that there's no error and the list of modules properly shows up:

```
Error: Please specify a valid module. It should be one of ["critical_css","render_blocking_js","page_cache","minify_js","minify_css","image_cdn","image_guide"]
```
